test/parallel/test-http2-respond-file-fd.js

```diff
@@ -1,19 +1,20 @@
 'use strict';
 
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 const http2 = require('http2');
 const assert = require('assert');
-const path = require('path');
 const fs = require('fs');
 
 const {
   HTTP2_HEADER_CONTENT_TYPE,
   HTTP2_HEADER_CONTENT_LENGTH
 } = http2.constants;
 
-const fname = path.resolve(common.fixturesDir, 'elipses.txt');
+const fname = fixtures.path('elipses.txt');
+
```

Comment:
Nit: This does not really need the newline IMO.

Reply:
Hi @tniessen I agree, removed now.